plugin/src/main/kotlin/com/ritense/plugin/service/PluginConfigurationSearchService.kt

```diff
@@ -0,0 +1,66 @@
+package com.ritense.plugin.service
+
+import com.ritense.plugin.domain.PluginActionDefinition
+import com.ritense.plugin.domain.PluginCategory
+import com.ritense.plugin.domain.PluginConfiguration
+import com.ritense.plugin.domain.PluginDefinition
+import javax.persistence.EntityManager
+import javax.persistence.TypedQuery
+import javax.persistence.criteria.CriteriaBuilder
+import javax.persistence.criteria.Join
+import javax.persistence.criteria.JoinType
+import javax.persistence.criteria.Predicate
+import javax.persistence.criteria.Root
+
+class PluginConfigurationSearchService(
```

Comment:
I know I wrote this, but sanity check: Is this actually a service, or a repository?

Reply:
Good quesion. It creates a query so it should be a repository